docs/manual/manual.tex

```diff
@@ -41,7 +41,8 @@
 \input{map-key-checker.tex}
 \input{interning-checker.tex}
 \input{lock-checker.tex}
-%\input{index-checker.tex}
+% Include index-checker.tex EXCEPT on buffalo.cs.washington.edu
+\IfFileExists{/scratch/secs-jenkins/java/jdk1.7.0}{\relax}{\input{index-checker.tex}}
```

Comment:
@mernst did you make this change?  Since I make the release on buffalo, this will cause the Index Checker to appear in the manual in the next release.